migration/src/lib/drupal-content.js

```diff
@@ -1,6 +1,16 @@
 import xml2js from 'xml2js';
 import { promises as fsPromises } from 'fs';
-import { blogMap } from "./maps.js";
+import { aboutMap } from "./maps-about.js";
```

Comment:
I'd say put the maps into a maps folder and then load from "maps/about.js" Just for cleaner organization.